update.js

```diff
@@ -133,11 +136,17 @@ var updater = {
     let lastWriteTime = 0;
     return text => {
       queue.push({text, time: new Date().toLocaleString()});
-      debounce(flushQueue, 1e3);
+      debounce(flushQueue, text && updater.checkAllStyles.running ? 1e3 : 0);
     };
     function flushQueue() {
       chromeLocal.getValue('updateLog').then((lines = []) => {
         const time = Date.now() - lastWriteTime > 11e3 ? queue[0].time + ' ' : '';
```

Comment:
1000 is a debounce delay.
11e3 is a timestamp coalescing delay.

Fixed the code by using 1000 to differentiate.